### ISSUE TITLE
The parameter groupName cannot be used with the parameter subnet

### ISSUE THREAD

**Issue by pditommaso** [nextflow-io/nextflow#465]
When creating a cloud cluster which configuration specifies a `securityGroup` and the `spotPrice` the following error message is raised 

```
ERROR ~ The parameter groupName cannot be used with the parameter subnet (Service: AmazonEC2; Status Code: 400; Error Code: InvalidParameterCombination; Request ID: xxxx-xxxx-xxxx-xxxx-xxxxxx)
```

**Comment by pditommaso** [comment]
Fixed in version `0.26.0-SNAPSHOT`. cc @aunderwo

**Comment by pditommaso** [comment]
To use it declare the following var in your env: 
```
export NXF_VER=0.26.0-SNAPSHOT
```

Make sure to refresh your snapshot running:
```
CAPSULE_RESET=1 nextflow info
```

then user NF as usual.

Then you can use NF
